Tasks/DeployAzureResourceGroup/task.json

```diff
@@ -202,8 +202,7 @@
         {
             "target": "resourceGroupName",
             "endpointId": "$(ConnectedServiceName)",
-            "dataSourceName": "AzureVirtualMachinesV2Id",
```

Comment:
I think you should introduce another binding. The original binding to list only VMs should be used in case of Select. I am not sure if Select action will work properly if RG has no VM in it. The create/delete actions can list all RGs and use a new binding for it. Thoughts?